hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMultiTargetMessageTask.java

```diff
@@ -56,7 +54,6 @@ protected void processMessage() throws Throwable {
         MultiTargetCallback callback = new MultiTargetCallback(targets);
         for (Address target : targets) {
             Operation op = operationFactory.createOperation();
-            op.setCallerUuid(endpoint.getUuid());
```

Comment:
Ah, this seems to be a backport of https://github.com/hazelcast/hazelcast/pull/9514